package.json

```diff
@@ -55,26 +55,27 @@
     "@metamask/eslint-config-jest": "^12.1.0",
     "@metamask/eslint-config-nodejs": "^11.0.1",
     "@metamask/eslint-config-typescript": "^11.0.0",
-    "@noble/hashes": "^1.1.3",
+    "@noble/hashes": "^1.3.1",
     "@types/jest": "^28.1.6",
-    "@types/node": "^17.0.23",
+    "@types/node": "^16.8.38",
     "@typescript-eslint/eslint-plugin": "^5.43.0",
     "@typescript-eslint/parser": "^5.43.0",
     "depcheck": "^1.4.3",
     "eslint": "^8.27.0",
     "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-import": "^2.26.0",
+    "eslint-plugin-import": "^2.27.5",
```

Comment:
We might not want to bump this yet, there seems to be a bug in 2.27.x: https://github.com/MetaMask/eslint-config/pull/307